packages/ui/src/components/Radio/radio.tsx

```diff
@@ -52,7 +53,9 @@ const Radio: IRadio = (props) => {
                     ? (
                         <div className={`${RADIO_CLASS_PREFIX}-label-container`}>
                             {wrappedIcon}
-                            <Typography.Text size='medium' disabled={disabled} {...labelProps}>{label}</Typography.Text>
+                            <Typography.Text size='medium' disabled={disabled} {...labelProps}>
+                                {label}{postfix && postfix}
```

Comment:
```
<Text>
    {string}{node}
</Text>
```
Seems strange:
1. How should you control disabled behaviour of `postfix`?
2. If we pass Nodes here, why not entire label is node?

Reply:
Made entire `label` as `React.ReactNode`